packages/framework/src/Component/Image/ImageUrlWithSizeHelper.php

```diff
@@ -2,19 +2,20 @@
 
 declare(strict_types=1);
 
-namespace Shopsys\FrameworkBundle\Model\Feed;
+namespace Shopsys\FrameworkBundle\Component\Image;
 
-class FeedItemImageHelper
+class ImageUrlWithSizeHelper
 {
     protected const DEFAULT_IMAGE_SIZE = 605;
 
     /**
      * @param string $imageUrl
      * @param int|null $width
+     * @param int|null $height
      * @return string
      */
-    public static function limitWidthInImageUrl(string $imageUrl, ?int $width = null): string
+    public static function limitWidthInImageUrl(string $imageUrl, ?int $width = null, ?int $height = null): string
     {
-        return $imageUrl . '?width=' . ($width ?? static::DEFAULT_IMAGE_SIZE);
+        return $imageUrl . '?width=' . ($width ?? static::DEFAULT_IMAGE_SIZE) . ($height !== null ? '&height=' . $height : '');
```

Comment:
I am not sure whether this would work as expected - there is no cropping so only one dimension is taken into account and the image is resized based on that (e.g. https://14-0.odin.shopsys.cloud/content/images/product/22-sencor-sle-22f46dm4-hello-kitty_1.jpg?width=605&height=605 does not result into a square image).

Moreover, the method name does not respect the behavior anymore (it only mentions width while there is also a `height` parameter). Let's discuss it, please

Reply:
btw, the cropping can be achieved like this:
https://14-0.odin.shopsys.cloud/content/images/product/22-sencor-sle-22f46dm4-hello-kitty_1.jpg?width=605&height=605&resize=fill

Reply:
I have renamed it

Also, I think there is no need to introduce cropping, just limiting size is enough

Reply:
ok